pkg/controlplane/instance_test.go

```diff
@@ -429,7 +429,7 @@ func TestNewBetaResourcesEnabledByDefault(t *testing.T) {
 		nodev1beta1.SchemeGroupVersion.WithResource("runtimeclasses"):                     true,
 		policyapiv1beta1.SchemeGroupVersion.WithResource("poddisruptionbudgets"):          true,
 		policyapiv1beta1.SchemeGroupVersion.WithResource("podsecuritypolicies"):           true,
-		storageapiv1beta1.SchemeGroupVersion.WithResource("csinodes"):                     true,
+		storageapiv1.SchemeGroupVersion.WithResource("csinodes"):                          true,
 	}
```

Comment:
We should only test  beta resources in `TestNewBetaResourcesEnabledByDefault`.

Reply:
test storageapiv1beta1 and storageapiv1